src/material-experimental/mdc-list/list-item.html

```diff
@@ -20,4 +20,5 @@
   Strong focus indicator element. MDC uses the `::before` pseudo element for the default
   focus/hover/selected state, so we need a separate element.
 -->
-<div class="mat-mdc-focus-indicator"></div>
+<div class="mat-mdc-focus-indicator"
+     [class.mat-mdc-focus-indicator-render]="_itemFocused"></div>
```

Comment:
It might just be me, but the class name sounds like "a render" of a focus indicator, rather than the indicator being rendered. Maybe the class can be called `focus-indicator-shown` or `focus-indicator-rendered`?